### ISSUE TITLE
All browsers and Word support floating images using the "align" attribute

### ISSUE THREAD

**Issue by FremyCompany** [SparkPost/heml#4]
Currently the safe HEML subset does not include any facility to float things on the side of a body of text, but floating images is widely supported as long as you use the "align" attribute (in particular Word supports this attribute). 

It would maybe be worth adding a css rewrite rule to support this?
https://twitter.com/FremyCompany/status/923992137791926272

**Comment by avigoldman** [comment]
Yeah, definitely worth doing as a CSS rule here, I'll write up a couple of test emails and open a PR on this.

**Comment by switch13** [comment]
Was this ever implemented? I could use it right now due to the fact that everything is left aligned and there doesn't seem to be a way to override that.
